content/de/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -9,7 +9,7 @@
 
 <body>
 
-<script src="https://katacoda.com/embed.js"></script>
+{{< katacoda-tutorial >}}
```

Comment:
This will actually render as an HTML comment. More importantly, though, using the shortcode triggers display of a notice about the removal.